app/src/components/market/common/market_scale/index.tsx

```diff
@@ -464,8 +464,16 @@ export const MarketScale: React.FC<Props> = (props: Props) => {
         ? Number(currentPrediction) * 100
         : (((startingPointNumber || 0) - lowerBoundNumber) / (upperBoundNumber - lowerBoundNumber)) * 100,
     )
-    setScaleValuePrediction(Number(newPrediction) * (upperBoundNumber - lowerBoundNumber) + lowerBoundNumber)
-  }, [newPrediction, currentPrediction, lowerBoundNumber, startingPointNumber, upperBoundNumber])
+    setScaleValuePrediction(calcPrediction(newPrediction?.toString() || '', lowerBound, upperBound, 18))
```

Comment:
hey I noticed we can prob cover this issue in this PR if you can be bothered: https://github.com/protofire/omen-exchange/issues/1626 

It is pretty much done, atm it is already displaying 0 instead of NaN.undefined, but I think we can just add the current prediction here to get the right starting value: `newPrediction?.toString() || currentPrediction?.toString() || ''`